### PR TITLE
Add missed comma in example pruning config

### PR DESCRIPTION
**Pull request by dkurt**
Also, there is an error when I try to use this config (v2.1.0):

```
jsonschema.exceptions.ValidationError: For algorithm: 'filter_pruning'
Additional properties are not allowed ('zero_grad' was unexpected)
```

### CODE REVIEW

**Comment by evgeniya-egupova** [comment]
> Also, there is an error when I try to use this config (v2.1.0):
> 
> ```
> jsonschema.exceptions.ValidationError: For algorithm: 'filter_pruning'
> Additional properties are not allowed ('zero_grad' was unexpected)
> ```

`zero_grad` parameter is no longer used and can be removed from docs.

BTW, this config shouldn't be treated as an example config as its purpose is to demonstrate all possible parameters and how you can change them. It also doesn't show the recommended parameter values. You can find real example configs in [examples folder](https://github.com/openvinotoolkit/nncf/tree/develop/examples)

**Comment by dkurt** [comment]
@evgeniya-egupova, thanks! Should I remove `zero_grad` to keep the document at least JSON compatible or we can close PR?

File: docs/compression_algorithms/Pruning.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ This approach allows pruning the model taking into account layer-specific sensit
         "prune_first_conv": false, // Whether to prune first Convolutional layers or not. First means that it is a convolutional layer such that there is a path from model input to this layer such that there are no other convolution operations on it. `False` by default (`True` by default in case of 'learned_ranking' interlayer_ranking_type).
         "prune_downsample_convs": false, // Whether to prune downsample Convolutional layers (with stride > 1) or not. `False` by default (`True` by default in case of 'learned_ranking' interlayer_ranking_type).
         "prune_batch_norms": true, // Whether to nullifies parameters of Batch Norm layer corresponds to zeroed filters of convolution corresponding to this Batch Norm. `True` by default.
-        "zero_grad": true // Whether to setting gradients corresponding to zeroed filters to zero during training, `True` by default.
+        "zero_grad": true, // Whether to setting gradients corresponding to zeroed filters to zero during training, `True` by default.
         "save_ranking_coeffs_path": "path/coeffs.json", // Path to save .json file with interlayer ranking coefficients.
```
Comment (evgeniya-egupova) [inline docs/compression_algorithms/Pruning.md:103]:
```suggestion
        "zero_grad": true, // Whether to setting gradients corresponding to zeroed filters to zero during training, `True` by default.
        "save_ranking_coeffs_path": "path/coeffs.json", // Path to save .json file with interlayer ranking coefficients.
```
```suggestion
        "save_ranking_coeffs_path": "path/coeffs.json", // Path to save .json file with interlayer ranking coefficients.
```

**Comment by evgeniya-egupova** [comment]
> @evgeniya-egupova, thanks! Should I remove `zero_grad` to keep the document at least JSON compatible or we can close PR?

Sure, you can remove `zero_grad`.

> @evgeniya-egupova, thanks! Should I remove `zero_grad` to keep the document at least JSON compatible or we can close PR?

Sure, you can remove `zero_grad`. I've added a suggestion to help with it.

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**